.github/eu_conan_setup.txt

```diff
@@ -0,0 +1,12 @@
+[settings]
```

Comment:
Please rename this file to something more informative e.g `ci_conan_profile.yml`.

Reply:
Renamed